File: Periode 3/Homework1/Exercise6/IntArrayOpperations.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Exercise6
{
    class IntArrayOpperations
    {
        private int[] List;

        public IntArrayOpperations(int[] list)
        {
            this.List = list;
        }

        public int Sum()
        {
            int output = 0;
            for (int i = 0; i < List.Count(); i++)
            {
                output = output + List[i];
            }
            //return List.Sum(); // <-- veel makkelijker
            return output;
        }

        public string Product()
        {
            string output = "";
            for (int i = 0; i < List.Count(); i++)
            {
                output = output + List[i];
            }
            
            return output;
        }
    }
}

File: Periode 3/Java/Chapter 1/1. Classes/Exercise6/src/IntArrayOpperations.java
/**
 * Created by Joost on 23-2-2017.
 */
public class IntArrayOpperations
{
    int[] I;

    public IntArrayOpperations(int[] i)
    {
        this.I = i;
    }

    public int Sum()
    {
        int output = 0;
        for(int i = 0; i < this.I.length; i++)
        {
            output += this.I[i];
        }
        return output;
    }

    public String Product()
    {
        String output = "";
        for(int i = 1; i < (this.I.length - 1); i++)
        {
            output += this.I[i];
        }
        return output;
    }
}
